tests/functional/includes/fn.bind.polyfill.js

```diff
@@ -0,0 +1,51 @@
+// Needed for PhantomJS < 2.0
+
+if (!Function.prototype.bind) {
```

Comment:
That is not really a polyfill, it returns a CJS module... It won't work as is with CasperJS.

I would much prefer using our own polyfill service for this and serve this polyfill as part of the page itself. Short of that, for this PR, I prefer to keep things as is.